### ISSUE TITLE
Keeps saying I'm signed out and won't accept input for new note

### ISSUE THREAD

**Issue by DavidBE-UK** [Garoe/ThunderKeepPlus#15]
So. I'm trying this for the first time, Windows 10, TB 78.4.0, ThunderKeepPlus 0.10.7. On clicking the Google Keep button in the TB toolbar, a Keep tab opens, but it only shows the first 7 notes, and a dialogue pops up saying 'You’ve been signed out. Please reload the page to sign in again and continue working.' with a Reload button and an X. If I click to reload I just get the same thing with the same dialogue each time. If I hit the X, the dialogue closes, but the tab still remains only showing the first 7 notes, and there's no reaction to me clicking into the Take A Note dialogue - it won't accept any input. According to the Google Account icon top right, though, I am actually signed in with my Google account despite what the dialogue says.

**Comment by Gara-Dorta** [comment]
Thanks for taking the time to report the issue. I've tested using the same setup as you and I'm unable to reproduce the issue, which unfortunately makes fixing it much more difficult. Could you try to do the following and see if that solves the issue?
1 Close the google keep tab
2 Clear cache: go into tools -> clear recent history -> change the "time range to clear" to "everything" and make sure "browsing history", "cookies" and "cache" are selected, and then click "clear now"
3 Reopen the google keep tab

**Comment by DavidBE-UK** [comment]
Aha. That's fixed it. It's all working properly now.

I notice that on opening the Google Keep tab after the cache clearing I initially got Google login pages in the new tab before the actual Keep page loaded. That didn't happen when I first opened it after first installing the extension: I just got what I described above. So presumably a cookie from Google from other Google usage in Thunderbird caused the proper login to be skipped after installing the extension.

Thanks - and thanks for the extension.
